backend/apps/ineedstudent/templates/zustimmung.html

```diff
@@ -3,35 +3,43 @@
 {% load i18n %}
 
 {% block title %}
-{% blocktrans %}Datenschutzerklärung Zustimmen{% endblocktrans %}
+    {% blocktrans %}Datenschutzerklärung Zustimmen{% endblocktrans %}
 {% endblock %}
 
 
 {% block header %}
-<link rel="stylesheet" href="{% static 'css/form.css' %}">
+    <link rel="stylesheet" href="{% static 'css/form.css' %}">
 
 {% endblock %}
 
 {% load crispy_forms_tags %}
 
 {% block content %}
 
-<div class="container">
-  <br>
-  <br>
-
-  <div class="card" style="margin-bottom:35px;">
-    <div class="card-header">
-      <h1 class="font-weight-light match4hc-title"> {% blocktrans %}Wir haben unsere Datenschutzerklärung Aktualisiert{% endblocktrans %} </h1>
-      <h2 class="font-weight-light match4hc-title">{% blocktrans %}Bitte stimmen Sie den aktualisierten Bedingungen zu{% endblocktrans %}</h2>
+    <div class="container">
+        <br>
+        <br>
+
+        <div class="card" style="margin-bottom:35px;">
+            <div class="card-header">
+                <h1 class="font-weight-light match4hc-title"> {% blocktrans %}Wir haben unsere Datenschutzerklärung
+                    Aktualisiert{% endblocktrans %} </h1>
```

Comment:
Small "A"

Reply:
You can directly offer this change and commit it ;)
```suggestion
                    aktualisiert{% endblocktrans %} </h1>
```

Reply:
But how?